### PR TITLE
Team3 [3pt] Implement dragon booking - Actions #50

### PR DESCRIPTION
**Pull request by MeqdamAlqudah**
1- Create reserve a dragon functionality.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,17 +11,15 @@ import Dragon from './components/Dragons/Dragon';
...
+      <Navigation />
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/missions" element={<MissionList />} />
+        <Route path="/Dragons" element={<Dragon />} />
+      </Routes>
+    </BrowserRouter>
+  </Provider>,
   document.getElementById('root'),
 );
```
Comment (zemola) [inline src/index.js]: Dear @MeqdamAlqudah ,

Kindly explain why you had to remove the **<React.StrictMode>** from index.js file.

**Review by zemola — CHANGES_REQUESTED** [review]
Hi @MeqdamAlqudah ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/index.js (hunk 1 above)
Comment (MeqdamAlqudah) [reply src/index.js]: we don't need React.StrictMode here but If you want me to keep it sure😀

**Review by zemola — APPROVED** [review]
Hi @MeqdamAlqudah ,

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

**Pull request merged**